netty/src/main/java/io/grpc/netty/NettyServerStream.java

```diff
@@ -61,6 +73,7 @@
     this.channel = checkNotNull(channel, "channel");
     this.http2Stream = checkNotNull(http2Stream, "http2Stream");
     this.handler = checkNotNull(handler, "handler");
+    this.attributes = buildAttributes(channel);
```

Comment:
Is it guaranteed that the channel is setup and TLS handshake has completed before a stream can be created?